<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreWindow</class>
 <widget class="QMainWindow" name="ScoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="enter_tag_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter tag:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="tag_input">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_team_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Register a new Team</string>
    </property>
    <property name="text">
     <string>Submit Tag</string>
    </property>
   </widget>
   <widget class="QListView" name="team_view">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mario Kart Lounge Team Score Tracker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="score_diff_display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="point_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter tag:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="point_input">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_point_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Allocate points to inputted team</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="race_label">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>310</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ICANT KEKW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="point_view">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Scores</string>
    </property>
   </widget>
   <widget class="QListView" name="limit_view">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="main_menu_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can also click ESC</string>
    </property>
    <property name="text">
     <string>Back to Main Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Current Race</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
